source/guide_olivetin.rst

```diff
@@ -0,0 +1,153 @@
+.. highlight:: console
+
+.. author:: Kim Diallo  <mail@diallo.kim>
+
+.. tag:: lang-nodejs
+.. tag:: audience-admins
+.. tag:: web
+.. tag:: console
+
+.. sidebar:: Logo
+
+  .. image:: ./_static/images/olivetin.png
+      :align: center
+
+##########
+OliveTin
+##########
+
+.. tag_list::
+
+OliveTin gives safe and simple access to predefined shell commands from a web interface.
+
+----
+
+.. note:: For this guide you should be familiar with the uberspace basic concepts of
+
+  * :manual:`web backends <web-backends>`
+  * :manual:`Node.js <lang-nodejs>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+OliveTin is released under the `AGPL-3.0 License`_.
+
+
+Installation
+============
+
+Download OliveTin
+------------------
+
+Check Github_ for the `latest release`_ and copy the download link to the linux-amd64.tar.gz file.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ wget --quiet https://github.com/OliveTin/OliveTin/releases/download/2022-04-07/OliveTin-2022-04-07-linux-amd64.tar.gz --output-document ~/olivetin.tar.gz
+ [isabell@stardust ~]$
+
+
+Extract OliveTin
+--------------------------
+
+Create a directory and unpack the archive into it.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mkdir ~/OliveTin
+ [isabell@stardust ~]$ tar -zxf olivetin.tar.gz --directory=$HOME/OliveTin --strip-components 1
+ [isabell@stardust ~]$
+
+
+Cleaning up
+-----------
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ rm olivetin.tar.gz
+ [isabell@stardust ~]$
+
+
+Configuration
+=============
+
+Configure OliveTin
+------------------
+
+Create a directory for the config file and copy the ``~/OliveTin/config.yaml`` into it.
+
+.. code-block:: ini
+
+ [isabell@stardust ~]$ mkdir ~/etc/OliveTin
+ [isabell@stardust ~]$ cp ~/OliveTin/config.yaml ~/etc/OliveTin/config.yaml
+
+Now you can add scripts and shell commands in '~/etc/OliveTin/config.yaml' which should be executed via the web interface. Use the examples in the file and the configdoc_ for orientation.
+
+Setup daemon
+------------
+
+Use your favourite editor to create ``~/etc/services.d/olivetin.ini`` with the following content:
+
+.. code-block:: ini
+
+ [program:olivetin]
+ directory=%(ENV_HOME)s/OliveTin
+ command=%(ENV_HOME)s/OliveTin/OliveTin --configdir %(ENV_HOME)s/etc/OliveTin
+ autostart=yes
+ autorestart=yes
```

Comment:
add startsecs=30

Reply:
If you have a better value for startsecs thats fine, but we should just add it at all so starting loops are prevented :-)